doc/source/install.rst

```diff
@@ -126,43 +126,3 @@ To build the documentation:
 .. code-block:: bash
 
     make -C doc html
-
-Installation on iOS
-===================
-
-`Pythonista <http://omz-software.com/pythonista/>`_ brings support for Python
```

Comment:
Pity that the (commercial) upstream seems to have not updated from 2020 and no longer provides a version of Python we support).